trajectory_executor/com_serial.py

```diff
@@ -39,6 +39,8 @@ def __init__(self):
 
   def sensor_data_request_callback(self, request: SensorDataRequest, response: SensorDataRequest):
     """Ping Arduino for sensor data"""
+    # self.get_logger().info("Requesting robot state")
+
```

Comment:
nit: debug comment?

Reply:
Yes and no. It's common practice (at least from what I've seen) in ROS to log information on what's going on in a node. This message however was getting spammed essentially so I left it there commented.

Reply:
kk cools makes sense. iirc since you are using a `logging` object, I believe you might be able to replace that with a debug statement and use set level to control the spam. i.e.

`self.get_logger().setLevel(logging.DEBUG)` or `self.get_logger().setLevel(logging.INFO)`

which then yields

```suggestion
    self.get_logger().debug("Requesting robot state")
```